src/components/material/material-buttons/online/MaterialButtonsOnlineInternal.tsx

```diff
@@ -111,6 +118,13 @@ const MaterialButtonsOnlineInternal: FC<MaterialButtonsOnlineInternalType> = ({
           variant="filled"
           size={size || "large"}
           dataCy={`${dataCy}-reader`}
+          trackClick={() =>
+            track("click", {
```

Comment:
I’m uncertain whether this should be a `"link"`. I’ll check with Tue before making any changes.